assets/airlines/cpz.json

```diff
@@ -1,15 +0,0 @@
-{
```

Comment:
Make sure to remove CPZ from the spawnPatterns of every airport they used to fly, otherwise those airports won't work